contracts/hw.device-type/iot-gate-imx8plus/contract.json

```
@@ -0,0 +1,39 @@
+{
+  "slug": "iot-gate-imx8plus",
+  "version": "1",
+  "type": "hw.device-type",
+  "aliases": [],
+  "name": "Compulab IOT-GATE-iMX8PLUS 4GB",
+  "assets": {
+    "logo": {
+      "url": "./iot-gate-imx8plus.svg",
+      "name": "logo"
+    }
+  },
+  "data": {
+    "arch": "aarch64",
+    "hdmi": false,
+    "led": false,
+    "connectivity": {
+      "bluetooth": true,
+      "wifi": true
+    },
+    "storage": {
+      "internal": true
+    },
+    "flashProtocol": "UMS",
```

```suggestion
    "flashProtocol": "usbMassStorage",
```

Nitpick to have full form instead of UMS